templateDataGenerator.go

```diff
@@ -524,3 +527,21 @@ func toYAML(v interface{}) string {
 	}
 	return string(data)
 }
+
+func renderToYAML(v interface{}, data interface{}) string {
+
+	value := toYAML(v)
+
+	tmpl, err := template.New("renderToYAML").Parse(value)
+	if err != nil {
```

Comment:
It fails silently, but I'd rather do that than possibly have existing deployments break next time they're deployed.

Reply:
Maybe add a log.Info in the future